doozer/tests/test_metadata.py

```diff
@@ -131,7 +133,7 @@ def _list_builds(self, builds, packageID=None, state=None, pattern=None, queryOp
             refined = [build for build in refined if build['state'] == state]
 
         refined.sort(key=lambda e: e['creation_ts'], reverse=True)
-        return refined
+        return refined[:1]
```

Comment:
we did not update this when we changed our brew query to return only 1 build, this updates that.